common-pg/src/main/scala/com/socrata/pg/soql/ExprSqlizer.scala

```diff
@@ -125,7 +125,7 @@ object ColumnRefSqlizer extends Sqlizer[ColumnRef[UserColumnId, SoQLType]] {
           escape: Escape): ParametricSql = {
     val tableMap = ctx(TableAliasMap).asInstanceOf[Map[String, String]]
     reps.get(QualifiedUserColumnId(expr.qualifier, expr.column)) match {
-      case Some(rep) if ctx(InnermostSoql) == true => // scalastyle:off simplify.boolean.expression
+      case Some(rep) if ctx(InnermostSoql) == true || expr.qualifier.nonEmpty => // scalastyle:off simplify.boolean.expression
```

Comment:
not inner most is handled in the next case statement.  Until joins, innermost is the only one that directly interacts with t tables.